cardano-api/src/Cardano/Api/Convert.hs

```diff
@@ -1,19 +1,28 @@
 module Cardano.Api.Convert
-  ( addressFromHex
+  ( ConversionError
+  , addressFromHex
   , addressToHex
+  , convertITNpublicKey
+  , convertITNsigningKey
```

Comment:
```suggestion
  , convertITNPublicKey
  , convertITNSigningKey
```

Reply:
Or better, stick to our terminology of verification key and signing key, rather than mixing it with public key and secret key, or a monster hybrid of public key and signing key ;-)